.github/workflows/test.yml

```diff
@@ -46,3 +46,34 @@ jobs:
     - name: Test flatbuffer conversions
       run: catkin test seerep_ros_conversions_fb
       shell: bash
+
+  integration-examples:
+    name: Integration Tests
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/agri-gaia/seerep_base:latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        path: src
+    - name: Set up python 3.8
+      run: |
+        apt update
+        apt install -y python3.8
+        ln -s `which python3.8` /usr/local/bin/python
+      shell: bash
+    - name: Build seerep
+      run: |
+        source /opt/ros/noetic/setup.bash
+        catkin build --workspace $GITHUB_WORKSPACE
+        source $GITHUB_WORKSPACE/devel/setup.bash
+      shell: bash
+    - name: Install python test dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox-gh-actions
```

Comment:
Using the [Setup Python Action](https://github.com/marketplace/actions/setup-python) would be a bit cleaner for setting up and installing the pip dependencies. However, nothing too important.

Reply:
I did some tests and the setup python action seems to mess with the `PYTHONPATH` environment variable and the python modules provided by the docker container are not recognized anymore. Catkin fails to build seerep and errors in some python module is not found. But this is just guess based on the symptoms.

Reply:
Ok then it's not worth the extra effort